inventory/all_projects/pulgit

```diff
@@ -0,0 +1,4 @@
+[pulgit_production]
```

Comment:
Can we call the VMs and groups `pulgitlab` or something? I feel like `git` is too generic, I'm not sure if it's a platform like github/gitlab, or some kind of git practice thing, or what.